deployment/provision_vagrant_vm.sh

```diff
@@ -36,21 +36,26 @@ service redis-server restart
 # install apache
 apt-get -q install -y apache2 apache2-dev libapache2-mod-wsgi-py3
 
-# With docker for mac, root will own the mount point (uid=0). chmod does not touch the host file system in these cases.
-OWNER=$(stat -c %U "$MOUNTPOINT/evap")
-if [ "$OWNER" == "root" ]; then chown -R 1042 "$MOUNTPOINT"; fi
+# link the mounted evap folder from the home directory:
+OWNER=$(stat -c %u "$MOUNTPOINT/evap")
+if [ "$OWNER" != 1042 ]; then
+  #  if, for any reason, vagrant failed to mount evap under the correct uid, use bindfs to link it to /opt/evap using the wanted user id
+  apt-get -q install -y bindfs
+  mkdir -p "$REPO_FOLDER"
+  bindfs --map="$OWNER/1042:@$OWNER/@1042" "$MOUNTPOINT" "$REPO_FOLDER"
```

Comment:
This will fail, if `/dev/fuse` is not explicitly forwarded to the docker container:
https://stackoverflow.com/a/49021109

Reply:
Mounting `/dev/fuse` is the responsibility of the user, as this only seems to apply to users who have a very nonstandard setup.

If you want to add local changes to your vagrantfile: https://github.com/fkoch-tgm/EvaP/wiki/Vagrant-with-libvirt#setup-for-fedora

Reply:
But you are right, it may be useful to note this somewhere :+1:

Reply:
adding an appropriate line in the `Vargentfile` may have following implications:
1. requires all docker users to install fuse
2. may require quirks on macos
Especially with 2. I am uncertain as to if it is a problem

Reply:
I thought it was only an issue for some docker users?

Reply:
okay, so in order to activate the quirks in the `Vagrantfile`, you need to manually add a `Vagrantfile.local`. Maybe we can echo a suggestion adding this, if the bindfs call fails here?

Reply:
It could be hard to spot that echo. Maybe we just want to `set -e` before `bindfs` so the script execution aborts there if the mount fails?

(We don't have `set -e` for the whole script so that, if a single niche setup aspect fails, it doesn't prevent the general setup to succeed, so new contributors can get a setup running even if e.g. the browser installation fails. However, if the mounting fails, I don't think we can provide an environment that's good for anything, so exiting would be fair)

Reply:
(or `mount-the-stuff || exit 1` instead of `set -e`)